rest-api/dao/physical_measurements_dao.py

```diff
@@ -340,9 +339,9 @@ def get_preferred_coding(codeable_concept):
     for coding in codeable_concept.coding:
       if pm_coding is None:
         pm_coding = coding
-      elif coding.system == _PM_SYSTEM:
-        if pm_coding.system == _PM_SYSTEM:
-          raise BadRequest('Multiple measurement codes with system %s' % _PM_SYSTEM)
+      elif coding.system.startswith(_PM_SYSTEM_PREFIX):
```

Comment:
This is so we accept both `physical-measurements` and `physical-evaluation`? If so... would be good to note this and add a TODO to lock down to the new value after the transition period is complete.

Reply:
This is intended to switch over to physical-measurements.

PMs using the existing system won't handle this; but we're planning on re-sending PMs for everybody, so I'm not sure it matters?

Reply:
This would just be about tightening our validation rules in the future, after the switchover. Your call!

Reply:
Ah. You mean explicitly reject something that uses physical-evaluation? Yeah, we could do that. Will consider for a follow-up.